doc/manual/manual.tex

```diff
@@ -6241,15 +6241,119 @@ \subsubsection{Crustal deformation}
 \end{figure}
 
 
+\subsubsection{Inner core convection}
+\label{sec:cookbooks-inner-core-convection}
 
+\textit{This section was contributed by Juliane Dannberg, and the model setup was insipired 
+by discussions with John Rudge.}
 
+This is an example for convection in the inner core of the Earth. It uses a spherical geometry 
+and is characterized by a single material with temperature dependent density, that makes it
+unstably stratified as temperature increases towards the center of the core. 
+The boundary conditions combine normal stress and normal velocity, and take into account the
+rate of phase change (melting/freezing) at the inner-outer core boundary. Gravity decreases 
+linearly from the boundary to zero at the center of the inner core. 
+The setup is analogous to the models described in \cite{Deguen2013}, and all material properties 
+are chosen in a way that the equations are non-dimensional.
 
+The required heating model and changes to the material model are implemented in a shared library
+(\texttt{inner\_core.cc}). In their non-dimensional form, the equations are
```

Comment:
you may actually include a url here, as in some of the other cookbooks, so that one can just click on it from the manual